packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -320,6 +320,9 @@ export class ColorPicker extends LitElement implements InteractiveComponent {
   /** When `true`, interaction is prevented and the component is displayed with lower opacity. */
   @property({ reflect: true }) disabled = false;
 
+  /** When `true`, hides the color graph. */
+  @property() fieldDisabled = false;
```

Comment:
This property should be reflected as well since its a boolean value property. A `reflects()` test can be added to ensure it does reflect too.

```tsx
 @property({reflect:true}) fieldDisabled = false;
```